src/mono/mono/eventpipe/ep-rt-mono.h

```diff
@@ -1321,21 +1321,11 @@ ep_rt_execute_rundown (void)
 #undef ep_rt_object_array_alloc
 #define ep_rt_object_array_alloc(obj_type,size) (g_new0 (obj_type, size))
 
-static
-inline
-void
-ep_rt_object_array_free (void *ptr)
-{
-	g_free (ptr);
-}
+#undef ep_rt_object_free
+#define ep_rt_object_free(obj_ptr) do { if (obj_ptr) g_free (obj_ptr); } while(0)
```

Comment:
`g_free (NULL)` is ok, there's no need for a check.

Reply:
correct, will change (the inlined method didn't have the check).